bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/ModuleType.java

```diff
@@ -219,7 +218,8 @@ public int[] getSignalLevels() {
     }
 
     public ModuleType getBridge() {
-        return bridgeType.orElse(UNKNOWN);
+        ModuleType bridgeType = this.bridgeType;
+        return bridgeType != null ? bridgeType : UNKNOWN;
```

Comment:
This can be a one-liner using `Objects.requireNonNullElse(..)`

Reply:
fixed